backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/placement/PlacementController.java

```diff
@@ -21,4 +17,10 @@ public PlacementController(PlacementService placementService) {
     public List<Placement> getAllPlacement() {
         return placementService.getAllPlacements();
     }
+
+    @PostMapping()
+    public Placement addNewPlacement(@RequestBody NewPlacementData requestPlacement) {
+        return placementService.addNewPlacement(requestPlacement);
```

Comment:
change requestPlacement to newRequestPlacement
to stick to the convention throughout the program